src/markdown-pages/install-with-kurl/removing-minio.md

```diff
@@ -0,0 +1,71 @@
+---
+path: "/docs/install-with-kurl/removing-minio"
+date: "2021-12-17"
+weight: 21
+linktitle: "Removing Minio"
+title: "Removing Minio from your Cluster"
+---
+
+As of [v2021.12.20-0](https://kurl.sh/release-notes/v2021.12.20-0), kURL clusters can be installed without add-ons having a peer dependency on object storage. 
+There is also a migration path for clusters to remove object storage, such as is the case with removing Minio.
+
+# New Installations
+
+The following add-ons can be installed without object storage:
+1. **Registry**: without an object store in the installer spec, a Persistent Volume (PV) will be used for storage. Note, that if the Rook  addon is installed, the Registry will ALWAYS use this as an object store.
+1. **Velero**: without object storage in the spec, no `default` BackupStorageLocation will be created. If the `disableS3` flag is set to `true` for the KOTS add-on, a PV-backed storage location will be created as the `default` location using the [Local-Volume-Provider](https://github.com/replicatedhq/local-volume-provider) plugin.
```

Comment:
Should default be in back-ticks or BackupStorageLocation? Is BackupStorageLocation a CR?

Reply:
`default` is the actual name of the object and BackupStorageLocation is the CR/type. @samantha-replicated what is the practice here?

Reply:
General best practice is to back-tick objects and CR in-line.